src/main/resources/application.properties

```diff
@@ -1 +0,0 @@
-
```

Comment:
I don't see any configuration here, is that correct? where are db conexion configurations for example?

Reply:
DB configuration should go here